### PR TITLE
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.12, and conda-forge-pinning 2021.10.06.08.16.35

### PR DESCRIPTION
**Pull request by datapythonista**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Ibis 2.0.0 release

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,22 +20,21 @@ requirements:
     - setuptools
 
   run:
+    - cached_property
```
Comment (xmnlab) [inline recipe/meta.yaml:23]: @datapythonista do we need to add dask as a dependency here?

**Comment by jreback** [comment]
@datapythonista merge when ready

**Comment by datapythonista** [comment]
@jreback do we want to have dask dependencies shipped with Ibis in conda? Or let's wait for when we have different packages for each backend?

**Comment by jreback** [comment]
i see we took out spark

it's ok to omit from the recipe i think

**Pull request merged**